apps/mailinglists/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+from django.utils.translation import ugettext as _
+
+
+class Mailinglist(models.Model):
+    email = models.EmailField(_("epostadresse"), unique=True)
+    name = models.CharField(max_length=100)
+    description = models.TextField(blank=True, null=True)
+    public = models.BooleanField(default=False)
+    contained_emails = models.ManyToManyField(
+        "self", blank=True, related_name="mailinglists_mail_is_in"
```

Comment:
Would love some input on what to name these fields, I am not convinced the current ones are clear enough to singnify what they are. (`contained_emails` == emails that are on this list, `mailinglists_mail_is_in` == self explainatory)

Reply:
I think it could be beneficial to separate the mailing lists and the mails that are in the list to separate models?
Having one model to represent the mailinglist itself with all its settings and such, and have another model to represent mails that are listed in it. Of course a thing can be both, but it simplifies the models a lot, w

Reply:
I tried splitting it up into two models now, would be great to have some feedback about potential solutions